tests/listobjects/listobjects_test.go

```diff
@@ -41,7 +37,26 @@ func testListObjects(t *testing.T, engine string) {
 	)
 	require.NoError(t, err)
 	defer conn.Close()
+	RunAllTests(t, pb.NewOpenFGAServiceClient(conn))
+}
+
+// RunAllTests will invoke all list objects tests
+func RunAllTests(t *testing.T, client ListObjectsClientInterface) {
+	t.Run("RunAll", func(t *testing.T) {
+		t.Run("ListObjects", func(t *testing.T) {
+			t.Parallel()
+			testListObjects(t, client)
+		})
+	})
+}
 
-	RunSchema1_1ListObjectsTests(t, pb.NewOpenFGAServiceClient(conn))
-	RunSchema1_0ListObjectsTests(t, pb.NewOpenFGAServiceClient(conn))
+func testListObjects(t *testing.T, client ListObjectsClientInterface) {
+	t.Run("Schema1_1", func(t *testing.T) {
+		t.Parallel()
+		RunSchema1_1ListObjectsTests(t, client)
+	})
+	t.Run("Schema1_0", func(t *testing.T) {
+		t.Parallel()
+		RunSchema1_0ListObjectsTests(t, client)
```

Comment:
Please unexport the 1.0 Check and ListObjects tests as we do not want anyone depending on them, especially given we are on the deprecation cycle for them. Technically we don't really need the 1.1 `RunSchema1_1...` tests to be exported as well, because we want to encourage usage of `RunAllTests` externally anyways.

Reply:
make runschema to be private